### DISCUSSION TITLE
Custom MP version numbering

### DISCUSSION THREAD

**Discussion by dreamer320** [micropython/micropython#11495] [General]
I would like to use my own MP version numbering for custom builds. On v1.19.1 I could do this by deleting .git and modifying the version=release line in docs/conf.py. However this doesn't work for v1.20.0, the version set in conf.py appears to be ignored.  Is there a way to override the standard version numbering for v1.20.0?
Thanks

**Comment by dlech** [comment]
This is now done by setting the `MICROPY_GIT_TAG` and `MICROPY_GIT_HASH` environment variables when generating the version header.

**Comment by dreamer320** [comment]
That works perfectly. Thanks
